src/composables/graph/useGraphNodeManager.ts

```diff
@@ -433,6 +433,28 @@ export function useGraphNodeManager(graph: LGraph): GraphNodeManager {
               })
           }
         }
+      } else if (
+        action === 'node:slot-errors:changed' &&
```

Comment:
Are you able to "instrument" this so that it comes through the same channel as the others?

Reply:
If we'd have to add the logic a second time for the slots, not worth it.

Reply:
this is a larger refactor... that's the thing i want to explore a bid.